src/Workspaces/Core/Portable/Shared/Extensions/ISymbolExtensions.cs

```diff
@@ -365,7 +368,8 @@ public static int GetArity(this ISymbol symbol)
             }
         }
 
-        public static ISymbol GetOriginalUnreducedDefinition(this ISymbol symbol)
+        [return: NotNullIfNotNull(parameterName: "symbol")]
```

Comment:
note: should we make 'rename' aware of the connection here?  i.e. if someone renames the aprameter, it would be good to have this be renamed.

Reply:
I believe the LDM was discussing changing the scope so nameof can be used against parameter symbols in this case, right? @jaredpar?